src/backend/marsha/core/admin.py

```diff
@@ -309,6 +309,37 @@ class VideoAdmin(BaseFileAdmin):
     verbose_name = _("Video")
 
 
+@admin.register(TimedTextTrack)
+class TimedTextTrackAdmin(admin.ModelAdmin):
+    """Admin class for the TimedTextTrack model."""
+
+    fields = (
+        "id",
+        "mode",
+        "extension",
+        "process_pipeline",
+        "video",
+        "language",
+        "uploaded_on",
+        "upload_state",
+        "created_on",
+        "updated_on",
+    )
+    readonly_fields = (
+        "id",
+        "mode",
+        "extension",
+        "process_pipeline",
+        "video",
+        "language",
+        "uploaded_on",
+        "upload_state",
+        "created_on",
+        "updated_on",
+    )
```

Comment:
```suggestion
    fields = (
        "id",
        "mode",
        "extension",
        "process_pipeline",
        "video",
        "language",
        "uploaded_on",
        "upload_state",
        "created_on",
        "updated_on",
    )
    readonly_fields = fields
```
Because the values of `readonly_fields` and `fields` are identical ?